boba-docs/docs/basics/6_deploy-contract.md

```diff
@@ -179,7 +185,27 @@ Summary
 ✨  Done in 10.11s.
 ```
 
-That's pretty much it. Contracts deployed! Tutorial complete. Hopefully now you know the basics of working with Boba!
+## Verify the Contract
+
+Using our tool [Bobascan](https://testnet.bobascan.com/), you can verify the contract has been deployed and is valid. Sign in by connecting your crypto wallet to Bobascan, then navigate to your transactions. [TODO: ADD IMAGES OF WHERE TO FIND TRANSACTIONS AND CONTRACT ADDRESS] This will lead you to the contract address of your newly deployed contract. Copy this address, then replace "`YOUR_CONTRACT_ADDRESS_HERE`" with it in the command below to verify your contract:
+
+```bash
+npx hardhat verify --network boba_sepolia YOUR_CONTRACT_ADDRESS_HERE
+```
+
+You should see the following output once you've successfully verified your contract:
+
+```console
+Nothing to compile
+Successfully submitted source code for contract
+contracts/YourContract.sol:YourContract at 0xFe0447Eecf08Eb16DfC5a3Fa2210B30dE3f2803d
```

Comment:
I don't know what your thought is on leaving deployment literals in the documenation.  Do you think we should? Or, should we mark this up with something like:

```
contracts/YourContract.sol:YourContract at 0x<address_of_your_contract_deployment>
```

Same for below